client/src/main/java/io/split/client/impressions/strategy/ProcessImpressionMTK.java

```diff
@@ -0,0 +1,25 @@
+package io.split.client.impressions.strategy;
+
+import io.split.client.impressions.Impression;
+import io.split.client.impressions.ImpressionsResult;
+import io.split.client.impressions.UniqueKeysTracker;
+
+import java.util.List;
+
+public class ProcessImpressionMTK implements ProcessImpressionStrategy{
+
+    private final UniqueKeysTracker _uniqueKeysTracker;
+
+    public ProcessImpressionMTK(UniqueKeysTracker uniqueKeysTracker) {
+        _uniqueKeysTracker = uniqueKeysTracker;
+    }
+
+    @Override
+    public ImpressionsResult processImpressions(List<Impression> impressions) {
+
+        for(Impression impression: impressions){
+            _uniqueKeysTracker.track(impression.split(),impression.key());
```

Comment:
When the sdk is in `none` mode, we also should to keep the total amount of evaluations. 
You should to add the impression count here as you are using in optimized mode.

Reply:
I added the impression count.